src/components/ConnectExchangeView/ExchangeAccountAdd/ExchangeAccountAdd.js

```diff
@@ -205,8 +207,6 @@ const ExchangeAccountAdd = ({ demo }) => {
                     id: "accounts.exchange",
                   })}
                   onChange={(/** @type {string} **/ v) => {
-                    setValue("exchangeType", typeOptions[0].val);
-                    setValue("testNet", false);
```

Comment:
Any reason you deleted that? I think we need to set to spot account when switching to kucoin for example, which doesn't have futures. That'll be an issue if previous selected exchange was binance with futures enabled.

Reply:
@cwagner22  this was only used for the cases when the exchange type selection was hidden, but now as that select input is always shown, these values are initialized automatically when the select input is rendered, we don't need them here, its sort of a duplication.

Reply:
For kucoin it's not shown since it's spot only.

Reply:
@cwagner22  for kucoin and bitmex, now its shown, the above checkI deleted, does that "typeOptions.length > 1", so now they are always shown, event its only one type.